gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageClientWriteChannelTest.java

```diff
@@ -111,121 +99,32 @@ public static void cleanUp() {
   @Test
   public void writeMultipleChunksSuccess() throws IOException {
     int numberOfChunks = 10;
-    writeChannel.initialize();
     ByteString data =
         GoogleCloudStorageTestHelper.createTestData(
             MAX_WRITE_CHUNK_BYTES.getNumber() * numberOfChunks);
     writeChannel.write(data.asReadOnlyByteBuffer());
     writeChannel.close();
     // Fake writer only writes half the buffer at a time
```

Comment:
`fakeWriteChannel` implementation only writes half a buffer at a time, we should not revert it with just one write call instead fix the code to validate that whole byte buffer is written properly.